_posts/2022-02-11-helm-multiple-worker-groups.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: How to run different worker types with the Dask Helm Chart
+author: Matthew Murray (NVIDIA)
+tags: [Kubernetes, Helm]
+theme: twitter
+---
+
+## Introduction
+Today, we’ll learn how to deploy Dask on a Kubernetes cluster with the Dask Helm Chart and then run and scale different worker types with annotations.
+
+### What is the Dask Helm Chart?
+The Dask Helm Chart is a convenient way of deploying Dask using Helm, a package manager for Kubernetes applications. After deploying Dask with the Dask Helm Chart, we can connect to our HelmCluster and begin scaling out workers.
```

Comment:
`Dask Helm Chart` should link to the chart

Reply:
`Helm` should link to the helm website.

Reply:
I would also not mention `HelmCluster` yet as you are about to introduce it in the next section. Perhaps `...connect to our Dask cluster and begin...` would be better.